jsapp/js/account/plans/plan.component.tsx

```diff
@@ -115,8 +114,7 @@ export default function Plan(props: PlanProps) {
   >([]);
   const [products, loadProducts, productsStatus] = useContext(ProductsContext);
   useRefreshApiFetcher(loadProducts, productsStatus);
-  const [organization, loadOrg, orgStatus] = useContext(OrganizationContext);
-  useRefreshApiFetcher(loadOrg, orgStatus);
+  const orgQuery = useOrganizationQuery();
```

Comment:
Wanna establish that we're gonna always use a QueryResult like this without destructuring?